frontend/kubecloud/src/views/ForgotPasswordView.vue

```diff
@@ -92,30 +103,24 @@ const handleRequestCode = async () => {
 }
 
 const handleVerifyCode = async () => {
+  if (!isCodeValid.value) return
+
   error.value = ''
   loading.value = true
   try {
-    // Get tokens from verification
+    // Verify the code and get tokens
     const tokens = await authService.verifyForgotPasswordCode({ email: email.value, code: Number(code.value) })
     authService.storeTokens(tokens.access_token, tokens.refresh_token)
     userStore.token = tokens.access_token
-    // Fetch user profile
-    const userRes = await api.get('/v1/user/', { requiresAuth: true, showNotifications: false }) as any
-    userStore.user = userRes.data.data.user
-    setTimeout(() => router.replace('/'), 1500)
+    // Redirect to reset password page
```

Comment:
i think we shouldn't store the token; we may store it to be used only for this page, `pass_token`.
as the user can skip this step as well.

Reply:
It should be stored to be used in the next step. forgot pass needs user to be authenticated.

Reply:
I stored token as a param to prevent any unauthed access.